selfhost/main.jakt

```diff
@@ -33,6 +33,9 @@ function help() -> String{
     output += "  -R,--runtime-path PATH\t\tPath of the Jakt runtime headers.\n\t\tDefaults to $PWD/runtime.\n"
     output += "  -o,--binary-dir PATH\t\tOutput directory for compiled files.\n\t\tDefaults to $PWD/build.\n"
     output += "  -C,--cxx-compiler-path PATH\t\tPath of the C++ compiler to use when compiling the generated sources.\n\t\tDefaults to clang++.\n"
+    output += "  -I PATH\t\tAdd PATH to compiler's include list. Can be specified multiple times.\n"
+    output += "  -L PATH\t\tAdd PATH to linker's search list. Can be specified multiple times.\n"
+    output += "  -l,--link-with LIB\t\tLink executable with LIB. Can be specified multiple times.\n"
```

Comment:
`-l` is used for the lexer's token debug stream (I'd suggest changing that one instead).